sample_app/bin/start_server

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+bundle install --jobs=4
+
+if [ -e /workspace/sample_app/tmp/pids/*.pid ]; then rm /workspace/sample_app/tmp/pids/*.pid; fi
+
+bundle exec rails db:prepare
+
+aws sqs --endpoint http://localstack:4566 create-queue --queue-name async-job-queue --no-cli-pager
+aws ses --endpoint http://localstack:4566 verify-email-identity --email $ACTION_MAILER_EMAIL
+
+bin/rails runner Aws::SessionStore::DynamoDB::Table.create_table
```

Comment:
I think it should be defined as rake task, not in migration [`lib/tasks/dynamo_db/session_store.rake`](https://github.com/aws/aws-sdk-rails/blob/940d543cb8bcacbe6a492c0a0d1f22324fef9d15/lib/tasks/dynamo_db/session_store.rake)

Reply:
Good idea. Would you like to create one? We're happy to take refactors.